src/admin.rs

```diff
@@ -162,19 +162,21 @@ fn dump_cert(x509: &X509) -> CertDump {
 
 async fn dump_certs(cert_manager: &SecretManager) -> Vec<CertsDump> {
     let mut dump = cert_manager
-        .collect_certs(|id, certs, state| {
+        .collect_certs(|id, certs| {
             let mut dump = CertsDump {
                 identity: id.to_string(),
-                state,
-                ca_cert: [CertDump {
-                    ..Default::default()
-                }],
-                cert_chain: Vec::new(),
+                ..Default::default()
+            };
+            use crate::identity::CertState::*;
```

Comment:
nit: what's the best practice in Rust for this?  most of the `use crate::` statements occur at the start of most of our files vs inline like above

Reply:
No idea. I put it here since it's used only here in this file and locally made sense and made the code shorter. If I were to put it at the global scope I'd probably have to use CertState instead (since names like "Available" feel pretty amibuous).